docs/understanding-your-pipeline/failed-events/index.md

```diff
@@ -16,15 +17,19 @@ All failed events are routed to storage (AWS S3 or GCP cloud storage).
 
 :::note
 
-When an error occurs while enriching an event (that is part of a collector payload with several events), the Failed Event will contain **only** this event. The other events of the collector payload can be successfully enriched.
+Collector might receive events in batches. So if something is wrong with the Collector payload as a whole, the generated failed event might represent several Snowplow events.
```

Comment:
> So if something is wrong with the Collector payload as a whole, the generated failed event might represent several Snowplow events.

I'd probably drop the "So" and say "the entire batch of Snowplow events" rather than just "several" which makes it sound like some may still be OK.
Should we give an example of some of the failure types that would impact the whole batch vs individual events? (Collector Payload Format Violation, Adaptor Failure, etc)